lib/active_admin/globalize/form_builder_extension.rb

```diff
@@ -5,12 +5,14 @@ module FormBuilderExtension
 
       def translated_inputs(name = "Translations", options = {}, &block)
         options.symbolize_keys!
+        available_locales = options.fetch(:available_locales, I18n.available_locales.sort)
         switch_locale = options.fetch(:switch_locale, false)
         auto_sort = options.fetch(:auto_sort, true)
```

Comment:
Since you gave user the option to choose locales to show (and implicitly the option to sort them), I'd remove the `auto_sort` option. Can you do that and write a note in readme?